python/lsst/validate/drp/io.py

```diff
@@ -0,0 +1,57 @@
+# LSST Data Management System
+# Copyright 2008-2016 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+
+from __future__ import print_function, division, absolute_import
+
+import json
+import numpy as np
+
+
+def saveKpmToJson(KpmStruct, filename):
```

Comment:
I think there was some prior conversation on this topic, which I appear to have missed, but I'm worried about 1) non-use of the Butler, 2) use of an explicit file format (what is the other side of this interface, if known?), and 3) use of an explicit file format other than YAML (effectively adding an additional dependency to the stack).